ppr-ui/src/components/mhrTransfers/SupportingDocuments.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+  <div id="supporting-documents" class="pb-3">
+    <p>
+      Select the supporting document you have for this owner:
+    </p>
+    <v-radio-group
+      id="supporting-docs-options"
+      v-model="deletedOwner.supportingDocument"
+      class="supporting-docs-options"
+      row
+      :disabled="isGlobalEditingMode"
+      hide-details="true"
+    >
+      <v-radio
+        id="supporting-doc-option-one"
+        :label="docOptions.optionOne.text"
+        active-class="selected-radio"
+        :value="docOptions.optionOne.value"
+        :ripple="false"
+      />
+      <v-radio
+        id="supporting-doc-option-two"
+        :label="docOptions.optionTwo.text"
+        active-class="selected-radio"
+        :value="docOptions.optionTwo.value"
+        :disabled="isSoleOwner"
+        :ripple="false"
+      />
+    </v-radio-group>
+    <div v-if="deletedOwner.supportingDocument === docOptions.optionOne.value"
+      class="supporting-doc-one">
+      <p>
+        <strong>Note:</strong> {{ docOptions.optionOne.note }}
+      </p>
+    </div>
+    <div v-if="deletedOwner.supportingDocument === docOptions.optionTwo.value"
+      class="supporting-doc-two">
+      <slot name="deathCert"></slot>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { useHomeOwners, useTransferOwners } from '@/composables'
+import { ApiTransferTypes, SupportingDocumentsOptions } from '@/enums/transferTypes'
+import { MhrRegistrationHomeOwnerIF } from '@/interfaces' // eslint-disable-line no-unused-vars
+import { defineComponent, reactive, toRefs, watch } from '@vue/composition-api'
+import { useActions } from 'vuex-composition-helpers'
+import { DeathCertificate } from '.'
+import { transferSupportingDocuments } from '@/resources/'
+
+export default defineComponent({
+  name: 'SupportingDocument',
+  props: {
+    deletedOwner: {
+      type: Object as () => MhrRegistrationHomeOwnerIF,
+      default: null
+    },
+    // Used to disable Death Cert when group has only one owner
+    isSoleOwner: {
+      type: Boolean
+    }
```

Comment:
Also super minor, just missing a default value here. 
With regards to the prop name, would it make sense to be more generic, not specific to the condition of the disabling, in the event we wanted to disable for another reason?

Reply:
Hmm, yea I see what you mean. Ok yea, I can rename.